internal/pkg/plugin/go_plugin_installer.go

```diff
@@ -16,13 +16,13 @@ type GoPluginInstaller struct {
 	Name string
 }
 
-func (g *GoPluginInstaller) CheckVersion(ver *version.Version) {
+func (g *GoPluginInstaller) CheckVersion(ver *version.Version) error {
 	versionCmd := exec.NewCommand("go", "version")
 
 	out, err := versionCmd.Output()
 	if err != nil {
 		output.ErrPrintf(programNotFoundMsg, "go")
-		return
+		return nil
```

Comment:
We should return an error here too, no?

Reply:
What I had in mind with this being a warning instead of an error is that they might have just forgotten to put it in their path.

But yeah, come to think of it, if `go` isn't found here then there's no way that the install command will succeed, and at that point they should be errors for all languages to be consistent.

I'll makethe change.